wp-content/themes/missilethreat/blocks/lazyblock-at-a-glance/block.php

```diff
@@ -0,0 +1,29 @@
+
+<?php
+/**
+ * At a Glance Block Template.
+ *
+ * @var  array  $attributes Block attributes.
+ * @var  array  $block Block data.
+ * @var  string $context Preview context [editor,frontend].
+ */
+
+
+$missile_short_name = get_field('missile_name');
+
+?>
+<div class="at-a-glance__wrapper alignwide">
+  <?php dynamic_sidebar( 'social-share' ); ?>
+  <div class="at-a-glance">
+    <h1 class="at-a-glance__title text--semibold">At a Glance</h1>
```

Comment:
I might add an option to rename this, with "At a Glance" being the default. Just in case.